Alignment/CommonAlignment/python/tools/trackselectionRefitting.py

```diff
@@ -272,6 +275,25 @@ def getSequence(process, collection,
                                              "clone": True})]
         if isCosmics: mods = mods[1:] # skip high purity selector for cosmics
 
+    #############################
+    ## PV Validation cuts tune ##                 
+    #############################
+
+    if isPVValidation:
+        options["TrackSelector"]["HighPurity"].update({
+                "trackQualities": [],
+                "pMin": 0.
+                })
+        options["TrackSelector"]["Alignment"].update({
+                "pMin" :      0.,      
+                "ptMin" :     0.,       
+                "nHitMin2D" : 0,       
+                "nHitMin"   : 0,       
+                "d0Min" : -999999.0,
+                "d0Max" :  999999.0,
+                "dzMin" : -999999.0,
+                "dzMax" :  999999.0
+                })
```

Comment:
@mmusich did you consider the altenative approach of using the common sequence first and overriding in your config as a second step similar to this:
```
process.refittingSequence = getSequence(...)
process.HighPurityTrackSelector.pMin = 0.0
...
```

This would avoid further cluttering of the common method, but I'm also fine with your approach.

Reply:
@ghellwig, that's exactly how it was before (https://github.com/cms-sw/cmssw/pull/21078/files/e8a5f259ec7056352d1c24bbb9c0ea5b2dba01f7#diff-72d8a78a6ccf3dc6cc176329075a4d3fL3) and I don't like it since the list of things to be changed is large.

Reply:
ok, as I said, I am also fine with this way